source/sass/buyers-guide/mixins.scss

```diff
@@ -69,9 +69,6 @@
 
 @mixin reset-active-nav-style() {
   .multipage-link.active {
-    color: $gray-60;
-    border-bottom: none;
-
```

Comment:
Looks like your PR accidentally removes this - please restore =)

Reply:
Sabrina wants the underline as an active state, this was preventing that so I removed it. Below is what it creates when I remove it (as the design requests). Though I can find another way around this with another css class if this causes too much trouble. 
![image](https://user-images.githubusercontent.com/4743971/98747169-3bb79e80-2374-11eb-9923-a5d5bc027714.png)

Reply:
Ignore my above comment. Reverted and applied styling a different want.